algebird-core/src/main/scala/com/twitter/algebird/BloomFilter.scala

```diff
@@ -225,6 +236,12 @@ case class BFInstance(hashes: BFHash, bits: BitSet, width: Int) extends BF {
       width)
   }
 
+  def checkAndAdd(item: String): (BF, ApproximateBoolean) = {
+    val bitsToActivate = BitSet(hashes(item): _*)
```

Comment:
let's not duplicate code here. Why not just use the default implementation of + below?